eng/pipelines/tasks/APIScan.yml

```diff
@@ -0,0 +1,32 @@
+steps:
+- task: DownloadPipelineArtifact@2
+  displayName: Download Pipeline Artifact
+  inputs:
+    source: "current"
+    path: "$(Pipeline.Workspace)"
+
+- task: CopyFiles@2
+  displayName: 'Copy Files to: $(Pipeline.Workspace)\ApiScanFiles'
+  inputs:
+    SourceFolder: '$(Pipeline.Workspace)\Lab.Release'
```

Comment:
This should be passed in as a parameter to this template file.

Reply:
May you clarify?